precompile/evolution-collection/src/mock.rs

```diff
@@ -67,23 +78,22 @@ impl sp_runtime::traits::Convert<AccountId, H160> for AccountIdToH160 {
 		account_id
 	}
 }
+pub struct H160ToAccountId;
+
+impl sp_runtime::traits::Convert<H160, AccountId> for H160ToAccountId {
+	fn convert(h160: H160) -> AccountId {
+		AccountId::from(h160)
+	}
+}
 
 impl pallet_laos_evolution::Config for Test {
 	type RuntimeEvent = RuntimeEvent;
 	type AccountIdToH160 = AccountIdToH160;
+	type H160ToAccountId = H160ToAccountId;
 	type MaxTokenUriLength = MaxTokenUriLength;
-}
-
-parameter_types! {
-	pub const ExistentialDeposit: u64 = 0;
-}
-#[derive_impl(pallet_balances::config_preludes::TestDefaultConfig as pallet_balances::DefaultConfig)]
-impl pallet_balances::Config for Test {
-	type Balance = Balance;
-	type ExistentialDeposit = ExistentialDeposit;
-	type AccountStore = System;
-	type RuntimeHoldReason = ();
-	type DustRemoval = ();
+	type WeightInfo = ();
```

Comment:
Another suggestion, no need to implement it in this PR. To avoid redeclaring types for every mock and runtime, you can use the [`with_default` macro](https://paritytech.github.io/polkadot-sdk/master/frame_support/pallet_macros/attr.config.html#optional-with_default) together with the [`TestDefaultConfig`](https://paritytech.github.io/polkadot-sdk/master/frame_system/pallet/config_preludes/struct.TestDefaultConfig.html#) in the `laos-evolution` pallet. There's an example in the polkadot-sdk: https://github.com/paritytech/polkadot-sdk/blob/3c213726cf165d8b1155d5151b9c548e879b5ff8/substrate/frame/sudo/src/lib.rs#L152-L185

Reply:
good point `with_default` is very handy ... let's try to use it in a refactor in a  new PR